Documentation/kubernetes-on-vagrant-single.md

```diff
@@ -49,6 +49,15 @@ Once Vagrant is finished booting and provisioning your machine, your cluster is
 
 Once in the `coreos-kubernetes/single-node/` directory, configure your local Kubernetes client using the following commands:
 
+You can choose from one of the two following options.
```

Comment:
There are really three options:

1. export `KUBECONFIG`
2. run the appropriate `kubectl config` commands
3. run kubectl with the `--kubeconfig` flag

Why do we need to document all of these methods?